md/wildfly-bundle.md

```diff
@@ -1,162 +1,178 @@
-The Wildfly bundle is a regular zip archive based on the Wildfly zip distribution.
+The WildFly bundle is a zip archive that contains the Red Hat WildFly JEE application server packaged with Bonita BPM Portal and [ready to use with Bonita BPM](BonitaBPM_platform_setup.md#platform_setup_tool).
+The WildFly bundle is a regular zip archive based on the WildFly zip distribution.
 
-## Installation of the Wildfly bundle
+## Installation of the WildFly bundle
 
-### Download and unzip the Wildfly bundle
+### Download and unzip the WildFly bundle
 
 <a id="download" />
 
 #### Download
 
 For the Community edition:
 
-* Go to the [Bonitasoft website](http://www.bonitasoft.com/downloads-v2) and get the Bonita BPM Community edition Wildfly bundle.
+* Go to the [Bonitasoft website](http://www.bonitasoft.com/downloads-v2) and get the Bonita BPM Community edition WildFly bundle.
 
 For a Subscription edition:
 
-* Go to the [Customer Portal](https://customer.bonitasoft.com/download/request) and download the Bonita BPM Subscription edition Wildfly bundle.
+* Go to the [Customer Portal](https://customer.bonitasoft.com/download/request) and download the Bonita BPM Subscription edition WildFly bundle.
 
 #### Unzip
 
-The folder where you unzip the Wildfly bundle is known as _`<WILDFLY_HOME>`_. We recommend the following locations: 
+The folder where you unzip the WildFly bundle is known as _`<WILDFLY_HOME>`_. We recommend the following locations:
 
 * Windows: `C:\BonitaBPM`. If you want to unzip the bundle to another folder, do not use spaces in the folder name. 
-* Linux: in `/opt/BonitaBPM`. Make sure that Linux user account used to execute Wildfly is the owner of the folders and files.
+* Linux: in `/opt/BonitaBPM`. Make sure that Linux user account used to execute WildFly is the owner of the folders and files.
 
-#### Content of the Wildfly bundle
+#### Content of the WildFly bundle
 
-The Wildfly bundle is based on a standard Wildfly installation with the following additions:
+The WildFly bundle is based on a standard WildFly installation with the following additions:
 
-* `bin/standalone.conf`: script to configure JVM system properties.
+* `bin/standalone.conf`: script to configure JVM system properties on Linux systems.
+* `bin/standalone.conf.bat`: script to configure JVM system properties on Windows systems.
 * `bonita-start.bat`: script to start the bundle on Windows.
 * `bonita-start.sh`: script to start the bundle on Linux.
-* `standalone/configuration/standalone.xml`: Wildfly context configuration for Bonita BPM Portal. It defines data sources used by Bonita BPM Engine.
+* `standalone/configuration/standalone.xml`: WildFly context configuration for Bonita BPM Portal. It defines data sources used by Bonita BPM Engine.
 * `request_key_utils`: script to generate license request keys (Subscription editions only).
 * `standalone/deployments/bonita-all-in-one-[version].ear`: Bonita BPM Portal (web application) and EJB3 API.
```

Comment:
It's not only the Portal but also Engine, replace with: "Bonita BPM web application and EJB3 API".